Tests/test_pairwise_aligner.py

```diff
@@ -1383,34 +1552,44 @@ def test_gap_here_only_1(self):
         # Very expensive to open a gap in seq1:
         nogaps = lambda x, y: -2000 - y  # noqa: E731
         # Very expensive to open a gap in seq2 unless it is in one of the allowed positions
-        specificgaps = lambda x, y: (-2 - y) if x in breaks else (-2000 - y)  # noqa: E731
+        specificgaps = (
+            lambda x, y: (-2 - y) if x in breaks else (-2000 - y)
+        )  # noqa: E731
```

Comment:
I would have expected ``# noqa: E731`` would need to be on the line above with the lambda, but flake8 is not complaining so that's probably fine as is.

Reply:
Yes, that's odd. In fact, I found that this is a bug: PyCQA/pycodestyle/issues/868

Reply:
Good detective work. We could remove the ``# noqa: E731`` (since it currently does nothing), or better move it up to the lambda line in the expectation that pycodestyle bug gets fixed later.

Reply:
Move it to the correct line.